### ISSUE TITLE
Could work with a single video?

### ISSUE THREAD

**Issue by Ruke805** [Thomasyse/BDMatch#7]
Is there a way to work with a single video only? I did a test here using a version that has a correct subtitle (DVD Rip) and synchronized correctly to another video (BD), but in many cases, there is no way to have a video that has a correct subtitle, a "TV version". Is there a way to just synchronize the subtitle with the version we want? Without using two videos? I tested it here and the program does not accept using only one video

**Comment by Thomasyse** [comment]
Well, I'm sorry that there's no solution in this project to synchronize the subtitle with a single video since it will be a much more complicated problem. However, there's a project working on this problem: [FFsubsync](https://github.com/smacke/ffsubsync#ffsubsync). Hope it will help.

**Comment by Ruke805** [comment]
Yeah, I see, I have tried FFsubsync, but it does not work well with Japanese audio for some reason

**Comment by Thomasyse** [comment]
Yeah, it's hard to make it work in all scenes. There's also some other tools introduced in its [Sync issues](https://github.com/smacke/ffsubsync#sync-issues) and maybe one of them will work.

**Comment by Ruke805** [comment]
Nice, I'll give a try
